agora/Agora/Governor.hs

```diff
@@ -66,6 +68,11 @@ data GovernorDatum = GovernorDatum
   -- ^ Gets copied over upon creation of a 'Agora.Proposal.ProposalDatum'.
   , nextProposalId :: ProposalId
   -- ^ What tag the next proposal will get upon creating.
+  , proposalTimings :: ProposalTimingConfig
+  -- ^ The timing configuration for proposals.
+  --   Will get copied over upon the creation of proposals.
+  , createProposalTimeRangeMaxDuration :: POSIXTime
+  -- ^ The maximum valid duration of a transaction that creats a proposal.
```

Comment:
Personally I would use "width" to describe it instead of duration. Plus, we could maybe newtype it?
```haskell
newtype MaxWidth 
  = MaxWidth { getMaxWidth :: POSIXTime }
```
We will likely reuse this in other places too.